ifrs17/Test/ImportStorageTest.ipynb

```diff
@@ -390,7 +390,46 @@
                 "\n    basicIfrsVariable with {AocType = AocTypes.WO, Value = 15.0},",
                 "\n  };",
                 "\n",
-                "\nvar activity = await CheckIfrsVariablesFromImportStorageAsync(ImportFormats.Actual, inputDataSetForWorkspace, inputDataSetForDataSource, ivsBenchmark);",
+                "\nvar activity = await CheckIfrsVariablesFromImportStorageAsync(ImportFormats.Actual, inputDataSetForWorkspace, inputDataSetForDataSource, ivsBenchmark, args);",
+                "\nactivity"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "activity.Status.Should().Be(ActivityLogStatus.Succeeded);"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "//There is a previous year, and for the current year Actuals are imported for a given Scenario",
```

Comment:
Looking at this file, it is beneficial to separate each test case in a subSection and extract the little comment we have at the beginning as a little description. Thanks

Reply:
done